app/code/Magento/Customer/Test/Unit/Block/Adminhtml/Edit/Tab/NewsletterTest.php

```diff
@@ -161,6 +168,55 @@ public function testInitFormCanNotShowTab()
         $this->assertSame($this->model, $this->model->initForm());
     }
 
+    /**
+     * Test getSubscriberStatusChangedDate
+     *
+     * @dataProvider getChangeStatusAtDataProvider
+     */
+    public function testGetSubscriberStatusChangedDate($statusDate, $dateExpected)
+    {
+        $customerId = 999;
+        $websiteId = 1;
+        $storeId = 1;
+        $isSubscribed = true;
+
+        $this->registryMock->method('registry')->with(RegistryConstants::CURRENT_CUSTOMER_ID)
+            ->willReturn($customerId);
+
+        $customer = $this->getMockForAbstractClass(CustomerInterface::class);
+        $customer->method('getWebsiteId')->willReturn($websiteId);
+        $customer->method('getStoreId')->willReturn($storeId);
+        $customer->method('getId')->willReturn($customerId);
+        $this->customerRepository->method('getById')->with($customerId)->willReturn($customer);
+
+        $subscriberMock = $this->getMockBuilder(Subscriber::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['loadByCustomer', 'getChangeStatusAt', 'isSubscribed', 'getData'])
+            ->getMock();
+        $this->localeDateMock->method('formatDateTime')->willReturn($statusDate);
```

Comment:
And ensure that the `getChangeStatusAt` value is provided to the formatDate
```suggestion
        $this->localeDateMock->method('formatDateTime')->with($statusDate)->willReturn($statusDate);
```